src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,18 @@
 import React from 'react';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard/MovieCard';
+import { Movie } from '../../types/Movie';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <div className="card" data-cy="Movie" key={movie.imdbId}>
+        <MovieCard {...movie} />
```

Comment:
The MoviesList should pass a movie to the MovieCard. 
And  destructure the movie inside the  MovieCard
```suggestion
        <MovieCard movie={movie} />
```